cfme/utils/rest.py

```diff
@@ -101,3 +103,19 @@ def create_resource(rest_api, col_name, col_data, col_action='create', substr_se
     # make sure action response is preserved
     rest_api.response = action_response
     return entities
+
+
+def delete_resources_from_collection(collection, resources, not_found=False, num_sec=10, delay=2):
+    collection.action.delete(*resources)
+    assert_response(collection._api)
+
+    for resource in resources:
+        resource.wait_not_exists(num_sec=num_sec, delay=delay)
+
+    if not_found or store.current_appliance.version < '5.9':
```

Comment:
**QUESTION**
Can we pass appliance into this fn and use that instead of `store.current_appliance.version`?